kakarot_scripts/constants.py

```diff
@@ -282,19 +282,32 @@ class ChainId(IntEnum):
     "UniversalLibraryCaller",
 ]
 
-# PRE-EIP155 TX
-MULTICALL3_DEPLOYER = "0x05f32b3cc3888453ff71b01135b34ff8e41263f2"
-MULTICALL3_SIGNED_TX = bytes.fromhex(
-    json.loads((DATA_DIR / "signed_txs.json").read_text())["multicall3"]
-)
-ARACHNID_PROXY_DEPLOYER = "0x3fab184622dc19b6109349b94811493bf2a45362"
-ARACHNID_PROXY_SIGNED_TX = bytes.fromhex(
-    json.loads((DATA_DIR / "signed_txs.json").read_text())["arachnid"]
-)
-CREATEX_DEPLOYER = "0xeD456e05CaAb11d66C4c797dD6c1D6f9A7F352b5"
-CREATEX_SIGNED_TX = bytes.fromhex(
-    json.loads((DATA_DIR / "signed_txs.json").read_text())["createx"]
-)
+PRE_EIP155_TX = {
+    "MULTICALL3": {
+        "deployer": "0x05f32b3cc3888453ff71b01135b34ff8e41263f2",
+        "signed_tx": bytes.fromhex(
+            json.loads((DATA_DIR / "signed_txs.json").read_text())["multicall3"]
+        ),
+        "address": "0xcA11bde05977b3631167028862bE2a173976CA11",
+        "required_eth": 0.1,
+    },
+    "ArachnidProxy": {
+        "deployer": "0x3fab184622dc19b6109349b94811493bf2a45362",
+        "signed_tx": bytes.fromhex(
+            json.loads((DATA_DIR / "signed_txs.json").read_text())["arachnid"]
+        ),
+        "address": "0x4e59b44847b379578588920ca78fbf26c0b4956c",
+        "required_eth": 0.1,
+    },
+    "CreateX": {
+        "deployer": "0xeD456e05CaAb11d66C4c797dD6c1D6f9A7F352b5",
+        "signed_tx": bytes.fromhex(
+            json.loads((DATA_DIR / "signed_txs.json").read_text())["createx"]
+        ),
+        "address": "0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed",
+        "required_eth": 0.3,
+    },
```

Comment:
since there is already a `signed_txs.json`, we can prob only do

```
PRE_EIP155_TXS = json.loads(...)
```

(possibly updating the json file)